geode-gfsh/src/integrationTest/java/org/apache/geode/management/internal/cli/commands/DescribeDiskStoreCommandIntegrationTest.java

```diff
@@ -98,4 +106,16 @@ public void commandSucceedsWithNameAndMember() throws Exception {
     gfsh.executeAndAssertThat(cmd).statusIsSuccess()
         .containsOutput(expectedData.toArray(new String[0]));
   }
+
+  @Test
+  public void testDirValidation() throws Exception {
```

Comment:
Can you remove `throws Exception` as it's not thrown from within the method body?. Since you're already modifying the file, also, it would be great if you can remove `throws Exception` from every single method in the class as none actually throws anything 👍 .